packages/eslint-plugin/src/rules/no-misused-spread.ts

```diff
@@ -57,8 +57,13 @@ export default createRule<Options, MessageIds>({
         'Using the spread operator on a Map in an object will result in an empty object. Did you mean to use `Object.fromEntries(map)` instead?',
       noPromiseSpreadInObject:
         'Using the spread operator on Promise in an object can cause unexpected behavior. Did you forget to await the promise?',
-      noStringSpread:
-        "Using the spread operator on a string can cause unexpected behavior. Prefer `.split('')` instead.",
+      noStringSpread: [
+        'Using the spread operator on a string can mishandle special characters, as can `split("")`.',
+        '- `...` produces Unicode code points, which will decompose complex emojis into individual emojis',
+        '- .split("") produces UTF-16 code units, which breaks rich characters in many languages',
+        "If this string does not need to preserve emojis or other non-English characters, disable this lint rule on this line or configure the 'allow' rule option.",
+        'Otherwise, consider using `Intl.Segmenter`.',
```

Comment:
Nice,
I would prefer these last two lines to be flipped, saying ~
Consider using Intl.Segmenter to split on graphemes
Otherwise, if this string does not need ....